src/librustc/hir/lowering.rs

```diff
@@ -5414,20 +5416,20 @@ impl<'a> LoweringContext<'a> {
         &mut self,
         span: Span,
         e: P<hir::Expr>,
-        args: hir::HirVec<hir::Expr>,
+        params: hir::HirVec<hir::Expr>,
```

Comment:
Whenever you have a call to a function, the expressions/values passed in are "arguments", so this (and other call-related functions below) should keep using "args".